wiki/osu!_wiki/Contribution_guide/zh.md

```diff
@@ -1,113 +1,66 @@
 
-   ![](img/compare-across-forks-no.jpg "不行。")
+2. 阅读 [最佳实践 § 做出更改](/wiki/osu!_wiki/Contribution_guide/Best_practices#making-changes)并进行需要的编辑。虽然你可以自由选择使用什么软件，但 osu! wiki 为下述两种编辑方式提供详细指南：
 
-5. 点击 `head fork` 下拉菜单，选择有你的用户名的那一项（一般为第二项）。
+  - [基于网页的 GitHub 内置的编辑器](/wiki/osu!_wiki/Contribution_guide/GitHub_web-based_editor) （需要联网，不需要安装）。
+  - [GitHub Desktop](/wiki/osu!_wiki/Contribution_guide/GitHub_Desktop) （可以离线，提供更多操作）。
 
-   ![](img/head-fork.jpg)
+## 自检
 
-6. 点击 `compare` 下拉菜单，选择你已经创建的那个分支，分支一般按字母排序。
+完成编辑后，请花一些时间校对你的工作。浏览以下快速清单：
 
-   ![](img/compare-branch.jpg)
+- **注意遣词**：编写 osu! wiki 文章时，除了特殊情况必须使用中性语气。
+- **风格与语法**：文章应该清晰、易于理解，并且不需要读者付出大量的脑力劳动。保持一致，避免过于复杂或突然的句子。使用带有内置拼写检查器的编辑器，例如 [Google 文档](https://docs.google.com)，以检查拼写错误和语法错误。
+- **[内容对等](/wiki/Article_styling_criteria/Formatting#content-parity)**：翻译必须包含与原始文章相同的信息（标点符号、措辞或格式的差异显然是允许的）。如果你认为原文的内容未完成、不准确或已过时，与其更改翻译的内容，你可以选择[提出一个 issues](https://github.com/ppy/osu-wiki/issues/new) 或者为原文章创一个新的拉取请求。
+- **结构与格式**：使用诸如 [jbt's Markdown Editor](https://jbt.github.io/markdown-editor/) 的工具来预览你的文章，以确保他看起来像你想要的样子。
+- **所有图像和其他非文本文件**必须[小于 1MB](/wiki/Article_styling_criteria/Formatting#file-size)。**截图**需要使用默认 osu! 皮肤和[特定设置](/wiki/Article_styling_criteria/Formatting#screenshots-of-gameplay)，同时，最大分辨率 1280×720。
 
-7. 点击 `Create pull request`。
+如需完整的规则列表，请阅读[文章风格规范](/wiki/Article_styling_criteria)。
 
-8. 输入英文的标题。对你更改的内容进行非常简短的解释。
+## [拉取请求（pull request）](https://docs.github.com/cn/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
 
-   对于文章翻译，在标题前面加上用方框括起来的两位语言代号，标题可以是你所翻译文章的名字。比如 `[FR] BBCode` 意味着 [BBCode](/wiki/BBCode) 这篇文章的法语翻译。
+在对更改进行仔细检查、提交并推送到你的分支后，你需要通过打开拉取请求将它们展示给 wiki 维护人员：
 
-9. 填写描述框中所需要的信息。你应该写上任何与 Pull Request 有关的信息，例如其完成程度已经你想让审阅者知道的任何信息。你也可以在描述中写入 "resolves #1" 或 "closes #1" 等关键字来让相关 issues 在这个 Pull Request 合并后自动关闭。（参见 Github Help 的文章 [Closing issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/)）
+1. 在**你的 osu! wiki 分支仓库**中，找到显示 `master↓` 的下拉菜单，然后选择包含更改的分支。
+2. 点击 `Contribute↓` 按钮并选择 `Open pull request`。
+3. 根据[最佳实践 § 创建拉取请求](/wiki/osu!_wiki/Contribution_guide/Best_practices#opening-a-pull-request) 然后点击 `Create pull request`。
 
-10. 完成后，点击 `Create pull request`。
+## 审阅
 
-    ![](img/new-pull-request.png)
+所有对 osu! wiki 的更改都需要审阅。在这个阶段，其他合作者会指出可能出现的错误以及修复它们的方法，无论任何形式，还是建议直接在拉取请求中编辑。和同行评审一样一样，你需要[应用他们的建议](/wiki/osu!_wiki/Contribution_guide/Best_practices#applying-reviews)或者解释你不想进行修改的原因。
 
-11. 下一步： [审阅（reviews）](#审阅) 与 [合并（merging）](#合并) 。
+如果一段时间内没有人检查你的文章，请尝试这样做：
 
-### 审阅
+- 确保你已解决所有建议 - 他们可能在等待你的回复。
+- 在 [osu!dev Discord server](/wiki/Community/osu!dev_Discord_server)（`#osu-wiki` 频道）或者 GitHub 的评论中询问其他 osu! wiki 编辑者。
+- 要获得翻译帮助，请检查几个你的语言的已合并拉取请求，并与他们的审阅者或者作者联系（[示例 GitHub 查询](https://github.com/ppy/osu-wiki/pulls?q=is%3Apr+is%3Amerged+%5BZH%5D)）。注：对于中文文章，你可以在 GitHub 的评论或描述中 `@osu-translate-zh` 来寻求 osu! 中文翻译组织 的审阅。
```

Comment:
My point about adding a comment that translates the extra sentence hasn't been addressed. Having a comment that points it out prevents it from being buried in obscurity.

Reply:
> My point about adding a comment that translates the extra sentence hasn't been addressed. Having a comment that points it out prevents it from being buried in obscurity.

```suggestion
- 要获得翻译帮助，请检查几个你的语言的已合并拉取请求，并与他们的审阅者或者作者联系（[示例 GitHub 查询](https://github.com/ppy/osu-wiki/pulls?q=is%3Apr+is%3Amerged+%5BZH%5D)）。注：对于中文文章，你可以在 GitHub 的评论或描述中 `@osu-translate-zh` 来寻求 osu! 中文翻译组织 的审阅。 <!-- On purpose deviated sentence: For article written in Chinese, contributors can link @osu-translate-zh in comment for review or/and help. -->
```

Should be ok?

Reply:
```suggestion
- 要获得翻译帮助，请检查几个你的语言的已合并拉取请求，并与他们的审阅者或者作者联系（[示例 GitHub 查询](https://github.com/ppy/osu-wiki/pulls?q=is%3Apr+is%3Amerged+%5BZH%5D)）。注：对于中文文章，你可以在 GitHub 的评论或描述中 `@osu-translate-zh` 来寻求 osu! 中文翻译组织 的审阅。<!-- Translation for additional sentence: "For articles written in Chinese, contributors can mention @osu-translate-zh in GitHub comments for review and/or help." -->
```